dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -711,3 +711,46 @@ def test_profile_merge_no_bin_overlap(self):
                                     'Profiles have no overlapping bin methods '
                                     'and therefore cannot be added together.'):
             profiler1 + profiler2
+
+    def test_profile_merge_with_different_options(self):
+        # Creating first profiler with default options
+        options = FloatOptions()
```

Comment:
what happens if we have multiple options? do we get two warnings? should we test for that?

Reply:
will order cause test to fail?

Reply:
It will print both warnings, but from what I can tell, it is not possible to test for more than one warning

Reply:
what do you mean by order?

Reply:
Fixed to handle multiple warnings